### DISCUSSION TITLE
sigmoid-azoth / obsidian-templates

### DISCUSSION THREAD

**Discussion by sigmoid-azoth** [SilentVoid13/Templater#310] [Templates Showcase]
Hi everyone,

Feel free to check out my template repo: https://github.com/sigmoid-azoth/obsidian-templates

**Comment by jroachgit** [comment]
Hey @sigmoid-azoth  - I'm trying to understand your template there, but I can't grasp it's purpose. Can you help explain or expand your Readme to include a purpose?

**Reply by jroachgit** [reply]
I'm not so new to obsidian, but new to a lot of these more adv functioning plugins.

**Comment by dylan-k** [comment]
link's broken
